presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoDriverUri.java

```diff
@@ -359,16 +389,23 @@ private static Properties mergeConnectionProperties(URI uri, Properties driverPr
         }
 
         Properties result = new Properties();
-        setProperties(result, defaults);
```

Comment:
I would prefer to have this code to work as it worked previously that defaults are overwritten instead of added later.